e2e_tests/conftest.py

```diff
@@ -1,24 +1,7 @@
-import os
-
 import pytest
-from django.core.management import call_command
 from playwright.sync_api import Page, BrowserContext
 
 
-from config.settings.base import BASE_DIR
-
-os.environ["DJANGO_ALLOW_ASYNC_UNSAFE"] = "true"
-
-
-@pytest.fixture(scope="session")
-def django_db_setup(django_db_setup, django_db_blocker):
```

Comment:
I know this does not work at the moment but we definitely need to fix (and not delete) this. Because otherwise: Firstly, some tests cannot be run repeatedly when they change the data. Secondly, the tests would depend on the currently running database whereas we would like to have a new database for each tests so that they do not affect each other.

Reply:
This actually prevented this PR from functioning, since overwriting fixture `django_db_setup` somehow interfered with the migration tests. That is the reason I removed it. It actually does nothing with respect to the e2e test. See my comment to #84.